robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -72,6 +72,10 @@ public class ShadowAccessibilityNodeInfo {
   private static final int EDITABLE_MASK = 0x00000080;
   
   private static final int TEXT_SELECTION_SETABLE_MASK = 0x00000100;
+  
+  private static final int CLEAR_A11YFOCUSABLE_MASK = 0x00000200;
```

Comment:
Yeah. Those aren't actions. They are properties. For actions, I feel like we should use the constants that already exist. For properties, we can either use booleans or create our own masks. If we're creating masks for our own implementation, though, they should be private.